packages/blocks/anodot/src/lib/get-user-accounts-action.ts

```diff
@@ -0,0 +1,33 @@
+import { createAction } from '@openops/blocks-framework';
+import { logger } from '@openops/server-shared';
+import { anadotAuth } from './anodot-auth-property';
+import { authenticateUserWithAnodot } from './common/auth';
+import { getAnodotUsers } from './common/users';
+
+export const getUserAccountsAction = createAction({
```

Comment:
I added this so I could do a loop over all accounts and then execute a custom API call